pkgs/development/compilers/crosstool-ng/fetcher.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, crosstool-ng
+, cacert
+, writeShellScript
+, preferLocalBuild ? true
+}:
+
+{ name
+, sha256
+  # use config from samples
+, configFromSample ? ""
+  # provide config content
+, configContent ? ""
+}:
+
+stdenv.mkDerivation {
+  inherit name;
+  builder = writeShellScript "builder.sh" ''
```

Comment:
I would be happier if we could use standard phases here

Reply:
Done!